crates/test-helpers/macros/build.rs

```diff
@@ -183,6 +184,106 @@ fn main() {
         }
     }
 
+    if cfg!(feature = "guest-c-utf16") {
```

Comment:
I think it's ok to skip an extra feature here, it's ok to gate the utf16 tests for C behind `guest-c`